components/nexus-commands/src/main/java/org/sonatype/nexus/commands/internal/InfoProviderMediator.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2008-2015 Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+
+package org.sonatype.nexus.commands.internal;
+
+import javax.inject.Named;
+
+import org.sonatype.sisu.goodies.common.ComponentSupport;
+
+import org.apache.karaf.shell.commands.InfoProvider;
+import org.eclipse.sisu.BeanEntry;
+import org.eclipse.sisu.Mediator;
+import org.osgi.framework.BundleContext;
+
+// FIXME: This still does not seem to do the trick, to get custom InfoProviders available in shell:info
```

Comment:
Sure if you can I'd like to understand why its not working, but its also not a huge deal if it just won't work for whatever reason, and we can just remove this.  But wanted to understand how to use Karaf and this was an extension point AFAICT so I think it kinda should work.  May point to some issues with the Sisu integration or other oddities which is really the only reason why I think a little investigation is still in order.